staking/programs/publisher-caps/src/lib.rs

```diff
@@ -222,6 +226,14 @@ pub struct VerifyPublisherCaps<'info> {
     pub encoded_vaa:    AccountInfo<'info>,
 }
 
+#[derive(Accounts)]
+pub struct ClosePublisherCaps<'info> {
+    pub write_authority: Signer<'info>,
+    #[account(mut, close = write_authority, has_one = write_authority @ PublisherCapsError::WrongWriteAuthority)]
```

Comment:
does `close = write_authority` mean that it will give the lamports to write_authority?

Reply:
added test